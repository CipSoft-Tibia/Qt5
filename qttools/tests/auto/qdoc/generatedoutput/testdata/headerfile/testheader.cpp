// Copyright (C) 2020 The Qt Company Ltd.
// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0

#include "testheader.h"

/*!
    \headerfile <TestHeader>
    \title Test Header
    \inmodule TestCPP
    \brief A header file.
    \ingroup headers
    \ingroup tests
*/

/*!
    \group headers
    \title Headers
*/

/*! \group tests
    \title Tests
*/

/*!
    \fn void globalFunc()
    \brief Global function.
    \relates <TestHeader>
*/

/*!
    \variable globalVar
    \brief Global variable.
    \relates <TestHeader>
*/

/*!
    \enum Globals
    \relates <TestHeader>
    \value Glo
    \value Bal
*/
