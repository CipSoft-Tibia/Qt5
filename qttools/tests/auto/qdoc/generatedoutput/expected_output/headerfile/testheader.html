<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
<!-- testheader.cpp -->
  <title>&lt;TestHeader&gt; - Test Header | HeaderFile</title>
</head>
<body>
<li><a href="headers.html">Headers</a></li>
<li>&lt;TestHeader&gt; - Test Header</li>
<div class="sidebar">
<div class="toc">
<h3><a name="toc">Contents</a></h3>
<ul>
</ul>
</div>
<div class="sidebar-content" id="sidebar-content"></div></div>
<h1 class="title">&lt;TestHeader&gt; - Test Header</h1>
<!-- $$$<TestHeader>-brief -->
<p>A header file. <a href="#details">More...</a></p>
<!-- @@@<TestHeader> -->
<div class="table"><table class="alignedsummary">
<tr><td class="memItemLeft rightAlign topAlign"> Header:</td><td class="memItemRight bottomAlign"> <span class="preprocessor">#include &lt;TestHeader&gt;</span>
</td></tr></table></div><ul>
</ul>
<a name="types"></a>
<h2 id="types">Types</h2>
<div class="table"><table class="alignedsummary">
<tr><td class="memItemLeft rightAlign topAlign"> enum </td><td class="memItemRight bottomAlign"><b><a href="testheader.html#Globals-enum">Globals</a></b> { Glo, Bal }</td></tr>
</table></div>
<a name="variables"></a>
<h2 id="variables">Variables</h2>
<div class="table"><table class="alignedsummary">
<tr><td class="memItemLeft rightAlign topAlign"> const int </td><td class="memItemRight bottomAlign"><b><a href="testheader.html#globalVar-var">globalVar</a></b></td></tr>
</table></div>
<a name="functions"></a>
<h2 id="functions">Functions</h2>
<div class="table"><table class="alignedsummary">
<tr><td class="memItemLeft rightAlign topAlign"> void </td><td class="memItemRight bottomAlign"><b><a href="testheader.html#globalFunc">globalFunc</a></b>()</td></tr>
</table></div>
<a name="details"></a>
<!-- $$$<TestHeader>-description -->
<div class="descr">
<h2 id="details">Detailed Description</h2>
</div>
<!-- @@@<TestHeader> -->
<div class="types">
<h2>Type Documentation</h2>
<!-- $$$Globals$$$Glo$$$Bal -->
<h3 class="fn" id="Globals-enum"><a name="Globals-enum"></a>enum <span class="name">Globals</span></h3>
<div class="table"><table class="valuelist"><tr><th class="tblConst">Constant</th><th class="tblVal">Value</th></tr>
<tr><td class="topAlign"><code>Glo</code></td><td class="topAlign tblval"><code>0</code></td></tr>
<tr><td class="topAlign"><code>Bal</code></td><td class="topAlign tblval"><code>1</code></td></tr>
</table></div>
<!-- @@@Globals -->
</div>
<div class="vars">
<h2>Variable Documentation</h2>
<!-- $$$globalVar -->
<h3 class="fn" id="globalVar-var"><a name="globalVar-var"></a>const <span class="type">int</span> <span class="name">globalVar</span></h3>
<p>Global variable.</p>
<!-- @@@globalVar -->
</div>
<div class="func">
<h2>Function Documentation</h2>
<!-- $$$globalFunc[overload1]$$$globalFunc -->
<h3 class="fn" id="globalFunc"><a name="globalFunc"></a><span class="type">void</span> <span class="name">globalFunc</span>()</h3>
<p>Global function.</p>
<!-- @@@globalFunc -->
</div>
</body>
</html>
